### ISSUE TITLE
Missing DLLs

### ISSUE THREAD

**Issue by Kerenok** [tothi/gcstar-win32#1]
Great initiative to publish this procedure and even provide a binary version of GCstar to help Windows users.

However, the executable throws messages complaining that LIBEAY32__.dll and libexpart-1__.dll are missing.

In the distribution directory generated in TEMP a file inc/lib/auto/Cairo/libexpat-1.dll is present though.

Could it be possible to include them with a compatible version in the executable?

**Comment by tothi** [comment]
of course. thx for calling attention. i didnt notice it because those dlls were available after building the exe, too. btw, i analyzed the dll imports and realized there are should be more missing dlls imported from the perl distrib. i'll try to include them in the exe build...

**Comment by Kerenok** [comment]
Thanks for the quick answer.
